tests/decorators/test_sensor.py

```diff
@@ -0,0 +1,131 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import pytest
+
+from airflow.decorators import task
+from airflow.exceptions import AirflowException, AirflowSensorTimeout
+from airflow.utils.state import State
+
+
+class TestSensorDecorator:
+    def test_sensor_fails_on_none_python_callable(self, dag_maker):
+        not_callable = {}
+        with pytest.raises(AirflowException):
+            task.sensor(not_callable)
+
+    def test_basic_sensor_success(self, dag_maker):
+        @task.sensor()
```

Comment:
```suggestion
        @task.sensor
```

Assuming the parathenses-less form works (I think it should since `task_decorator_factory` provides that?) this should be used for consistency. (Although I personally dislike it.)

Reply:
Fixed.